content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: Prow and Tide for Kubernetes Contributors
+date: 2022-05-31
+slug: prow-and-tide-for-kubernetes-contributors
+---
+
+**Authors:** [Chris Short](https://github.com/chris-short)
+
+---
+
+In my work in the Kubernetes world, I look up a label or Prow command often. The systems behind the scenes ([Prow](https://prow.kubernetes.io/) and [Tide](https://pkg.go.dev/k8s.io/test-infra/prow/cmd/tide#section-readme)) are here to help Kubernetes Contributors get stuff done.
+
+Labeling which SIG, WG, or subproject is as important as the issue or PR having someone assigned. To quote [the docs](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md), "Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md) component for managing a pool of GitHub PRs that match a given set of criteria. It will automatically retest PRs that meet the criteria ('tide comes in') and automatically merge them when they have up-to-date passing test results ('tide goes out')."
+
+Something that really helps move things along is the [label_sync](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#intro) tool along with the logic Prow and Tide bring to the community.
+
+What actually prompted this article is the awesomely amazing folks on the Contributor Comms team saying, "I need to squash my commits and push that." Which immediately made me remember the wonder of the Tide label: [`tide/merge-method-squash`](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#tide/merge-method-squash)
+
+That's right. Squashing your commits is a label away and the tooling will do the rest. To do this on your PR you'll need to comment with the following:
+
+`/label tide/merge-method-squash`
+
+These two pages document all the functionality available to Kubernetes contributors (either through labels or Prow). The biggest thing to remember is when a something is a label or a command:
+
+* [Prow Command Help](https://prow.kubernetes.io/command-help)
+* [test-infra/label_sync/labels.md](https://github.com/kubernetes/test-infra/tree/master/label_sync) (remember, all labels are preceded with `/label`)
```

Comment:
This is not quite correct. Most labels are applied using their particular command like `/area <foo>` or `/sig <blah>`
Only labels that don't have an associated command are applied with the `/label` command.